aspnetcore/host-and-deploy/iis/modules.md

```diff
@@ -15,75 +15,75 @@ uid: host-and-deploy/iis/modules
+| **URL Authorization**<br>`UrlAuthorizationModule` | Yes | [ASP.NET Core Identity](xref:security/authentication/identity) |
+| **Windows Authentication**<br>`WindowsAuthenticationModule` | Yes | |
+
+&#8224;The URL Rewrite Module's `isFile` and `isDirectory` don't work with ASP.NET Core apps due to the changes in [directory structure](xref:host-and-deploy/directory-structure).
 
 ## Managed Modules
 
-Module | .NET Core Active | ASP.NET Core Option
---- | :---: | ---
-AnonymousIdentification | No | 
-DefaultAuthentication | No | 
-FileAuthorization | No | 
-FormsAuthentication | No | [Cookie Authentication Middleware](xref:security/authentication/cookie)
-OutputCache | No | [Response Caching Middleware](xref:performance/caching/middleware)
-Profile | No | 
-RoleManager | No | 
-ScriptModule-4.0 | No | 
-Session | No | [Session Middleware](xref:fundamentals/app-state)
-UrlAuthorization | No | 
-UrlMappingsModule | No | [URL Rewriting Middleware](xref:fundamentals/url-rewriting)
-UrlRoutingModule-4.0 | No | [ASP.NET Core  Identity](xref:security/authentication/identity)
-WindowsAuthentication | No | 
+| Module                  | .NET Core Active | ASP.NET Core Option |
+| ----------------------- | :--------------: | ------------------- |
+| AnonymousIdentification | No               | |
+| DefaultAuthentication   | No               | |
+| FileAuthorization       | No               | |
+| FormsAuthentication     | No               | [Cookie Authentication Middleware](xref:security/authentication/cookie) |
+| OutputCache             | No               | [Response Caching Middleware](xref:performance/caching/middleware) |
+| Profile                 | No               | |
+| RoleManager             | No               | |
+| ScriptModule-4.0        | No               | |
+| Session                 | No               | [Session Middleware](xref:fundamentals/app-state) |
+| UrlAuthorization        | No               | |
+| UrlMappingsModule       | No               | [URL Rewriting Middleware](xref:fundamentals/url-rewriting) |
+| UrlRoutingModule-4.0    | No               | [ASP.NET Core  Identity](xref:security/authentication/identity) |
+| WindowsAuthentication   | No               | |
 
 ## IIS Manager application changes
 
-Using IIS Manager to configure settings, the *web.config* file of the app is changed. If deploying an app and including *web.config*, any changes made with IIS Manger are overwritten by the deployed *web.config* file. If changes are made to the server's *web.config* file, copy the updated *web.config* file to the local project immediately.
+When using IIS Manager to configure settings, the *web.config* file of the app is changed. If deploying an app and including *web.config*, any changes made with IIS Manger are overwritten by the deployed *web.config* file. If changes are made to the server's *web.config* file, copy the updated *web.config* file on the server to the local project immediately.
```

Comment:
Manger --> Manager

Reply:
:alien: ........... I did *not* do that. :smile: